api/rs/slint/lib.rs

```diff
@@ -316,14 +319,19 @@ macro_rules! include_modules {
 /// }
 /// ```
 ///
-/// For example, assuming this is in a crate called example, and the default locale
-/// is configured to be french, it will load translation at runtime from
-/// `/path/to/example/translations/fr/LC_MESSAGES/example.mo`
+/// For example, assuming this is in a crate called `example` and the default locale
+/// is configured to be French, it will load translations at runtime from
+/// `/path/to/example/translations/fr/LC_MESSAGES/example.mo`.
+///
+/// Another example of loading translations relative to the executable:
+/// ```rust
+/// slint::init_translations!(std::env::current_exe().unwrap().parent().unwrap().join("lang"));
```

Comment:
Suggestion to use the same path suffix as in the above example.

```suggestion
/// slint::init_translations!(std::env::current_exe().unwrap().parent().unwrap().join("translations"));
```